### PR TITLE
Configure mongo db

### PR DESCRIPTION
**Pull request by vaniisgh**
move towards resolving  #3 and #2

### CODE REVIEW

File: mock_drs/database/data_objects.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+      "type": "ftp",
+      "access_url": {
+        "url": "ftp://ftp.ensembl.org/pub/release-96/fasta/homo_sapiens/dna//Homo_sapiens.GRCh38.dna.chromosome.19.fa.gz",
+        "headers": {
+          "Authorization": "None"
+        }
+      }
+    }
+  ]
+}
```
Comment (uniqueg) [inline mock_drs/database/data_objects.json]: As GitHub indicates, there should be a newline at the end of the file

Comment (vaniisgh) [reply mock_drs/database/data_objects.json]: I didn't realize that's what it means, will take note. thanks

File: mock_drs/database/register_mongodb.py [hunk 2]
```diff
@@ -2,78 +2,67 @@
 
 import logging
 from typing import Dict
+import json
```
Comment (uniqueg) [inline mock_drs/database/register_mongodb.py]: Try to keep the following order for imports:
- use three import blocks:
  1. standard Python modules
  2. third party Python modules
  3. modules within _this_ package
- within each block, arrange modules in alphabetical order (e.g., `json`, `logging`, `typing`)

File: mock_drs/database/register_mongodb.py [hunk 3]
```diff
@@ -2,78 +2,67 @@
...
 
-#from app.config.config_parser import get_conf
-#from app.ga4gh.drs.endpoints.get_service_info import get_service_info
+import pymongo
 
+from config.config_parser import get_conf
 
-# Get logger instance
-logger = logging.getLogger(__name__)
+from random import choice
```
Comment (uniqueg) [inline mock_drs/database/register_mongodb.py]: Put _all_ imports in the import blocks at the top. There has to be a very good reason why imports should not be at the top.

Comment (vaniisgh) [reply mock_drs/database/register_mongodb.py]: will fix this.

**Comment by uniqueg** [comment]
And now we have conflicts because the same files have been changed in different branches since they were forked from `master` :) No problem, just saying for the future

**Comment by uniqueg** [comment]
Can you resolve the conflicts? If you click on the button it will open an editor and indicated the conflicting areas of the code. Make sure to keep what you want to keep after merging and delete the _other_ version. Also delete all the markup which indicated from which branch which version comes. Then please ask me again to review

**Pull request merged**